src/Hilke.KineticConvolution/Hilke.KineticConvolution.csproj

```diff
@@ -22,6 +22,7 @@
   <ItemGroup>
     <PackageReference Include="Fractions" Version="5.0.1" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="morelinq" Version="3.3.2" />

```

Comment:
I'd like to see this dependency go away. There's already an implementation of `CyclicPairwise` in the Extensions folder you can take inspiration from.